### PR TITLE
Update to overlap logic 

### PR DESCRIPTION
**Pull request by alvak1**
Validity Start date/time and end date/time should be considered to check for overlapping validities. Ending on date for repeating disruptions are not part of overlap check

### CODE REVIEW

File: site/pages/api/create-disruption.test.ts [hunk 1]
```diff
@@ -622,126 +611,4 @@ describe("create-disruption API", () => {
         );
         expect(writeHeadMock).toBeCalledWith(302, { Location: `/create-disruption/${defaultDisruptionId}` });
     });
-
-    it("should confirm that the dates are expanded as expected", () => {
```
Comment (RobCattonKPMG) [inline site/pages/api/create-disruption.test.ts:626]: Why are we deleting all these tests?

File: site/schemas/create-disruption.schema.ts [hunk 2]
```diff
@@ -499,33 +499,6 @@ export const createDisruptionsSchemaRefined = createDisruptionSchema
             ];
 
             let valid = true;
```
Comment (RobCattonKPMG) [inline site/schemas/create-disruption.schema.ts:501]: What is valid used for now?

File: site/pages/api/create-disruption.test.ts (hunk 1 above)
Comment (alvak1) [reply site/pages/api/create-disruption.test.ts:626]: The test case was for expandedValidity function. As this function is deleted the test case would redundant

File: site/schemas/create-disruption.schema.ts (hunk 2 above)
Comment (alvak1) [reply site/schemas/create-disruption.schema.ts]: isValid is removed

**Review by RobCattonKPMG — APPROVED** [review]

**Pull request merged**